smarts/core/trap_manager.py

```diff
@@ -45,18 +45,21 @@ class Trap:
     reactivation_time: float
     remaining_time_to_reactivation: float
     patience: float
+    elapsed_sim_time: float = 0
 
-    def step_trigger(self, dt: float):
-        self.remaining_time_to_reactivation -= dt
+    def update_sim_time(self, elapsed_sim_time: float):
+        self.elapsed_sim_time = elapsed_sim_time
```

Comment:
I think with this `remaining_time_to_reactivation` no longer maintains its original meaning. I think this also breaks the ability for the trap to reset and hijack another vehicle. Although I am not sure the functionality was ever used. This is fine for now but put up an issue for this leftover to either remove the functionality or fix it.

Reply:
`reactivation_time` is assigned to -1, which means for now it would not be reused, right?

Reply:
Yes sorry, it is fine, I do not think it is used at current. Reactivation should be stripped out as part of https://github.com/huawei-noah/SMARTS/issues/63.